cloudcix_primitives/gpu_lxd.py

```diff
@@ -19,6 +19,7 @@ def build(
         project: str,
         instance_name: str,
         device_identifier: str,
+        gpu_name: str,
```

Comment:
```suggestion
        device_name: str,
```

As discussed, can this be `device_name` and then by removing line 66 there should be no further changes. Narendra has already prepared robot to pass `device_name` in the gpu_lxd primitive calls.